src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.spring.model.dto.request;
+
+import com.sun.istack.NotNull;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.Size;
+
+public class UserRequestDto {
+    @NotNull
```

Comment:
this code doesn't relate to current HW, let's not push currently unneeded code in future